RecoLocalTracker/SiStripZeroSuppression/python/customiseHybrid.py

```diff
@@ -0,0 +1,37 @@
+import FWCore.ParameterSet.Config as cms
+
+## Adjust the local reco sequence for running on hybrid zero-suppressed data
+def runOnHybridZS(process):
```

Comment:
do I understand correctly that this customization is needed only for running on the emulated hybrid data, created with ```addHybridEmulationBeforeRepackZS10``` defined below in this file?
Please add a note about this in the comments.
If so, it should be OK to have this as a modifier.

However, if this change will be required for real data taking, then we have to use process modifiers.

Reply:
Yes and no: this is what is needed to run on hybrid data (emulated or not - that difference is covered by `--repacked`). We won't need it in RECO for real data taking, but what we will need in the HLT when doing the repacking there should be essentially this as well (to be tested).

Reply:
uhm, instead of the shortcut that we have now, then I'd much rather that we have this modification and HLT-like repacking done in a separate cmsDriver step so that the RECO part is actually what will be running in data taking. This is in particular to be sure that we are not missing anything.

Reply:
So you mean step1 hybrid emulation & repack, step2 ZS & repack and step3 RECO (unmodified)? That makes sense, I'll try to get it to work.

Reply:
yes, 3 steps.
thank you.